openapi/openapi-service.ytt.yaml

```diff
@@ -0,0 +1,30 @@
+#@ load("openapi.lib.yml", "response", "reference", "type", "array")
+
+paths:
+  /telemetry:
+    get:
+      summary: Collect telemetry data
+      description: Collect telemetry data including app info, system info, collections info, cluster info, configs and statistics
+      operationId: telemetry
```

Comment:
could you please add tags, so the API is grouped in UI properly

Reply:
done